phalcon/db/adapter/pdo.zep

```diff
@@ -85,9 +85,12 @@ abstract class Pdo extends Adapter
 	 */
 	public function connect(descriptor = null)
 	{
-		var username, password, dsnParts, dsnAttributes,
+		var eventsManager,
+			username, password, dsnParts, dsnAttributes,
 			persistent, options, key, value;
 
+		let eventsManager = <ManagerInterface> this->_eventsManager;
```

Comment:
This is another idea, but what if someone don't want to use eventsManager from di and create seperated ? Correct me if i'm wrong, we can basically have one eventsManager for whole application for watching all fire events and dispatch them to listeners, this will mean that we will have less memory usage but i think this will cost more time(beacause it will cost more time to find proper listener when we will have a lot of them) or we can have many eventsManagers which will cost more memory but will be faster ?